lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -925,17 +986,35 @@ TotalCreditUsage SessionState::get_total_credit_usage() {
   return usage;
 }
 
-uint32_t SessionState::get_local_teid() const {
-  return local_teid_;
+std::string SessionState::get_subscriber_id() const {
```

Comment:
Is this not the same as `get_imsi` ? https://sourcegraph.com/github.com/magma/magma/-/blob/lte/gateway/c/session_manager/SessionState.h#L253

Reply:
Yes. Same as get_imsi(). I removed the get_subscriber_id() and used the get_imsi().